### PR TITLE
Development farmingrewards + planetpositive

### PR DESCRIPTION
**Pull request by Sacha96**
Planet Positive Farming + remove duplication of information

### CODE REVIEW

**Review by andhartl — COMMENTED** [review]
'CPR Reward can change over time' is misleading. should say 'CPR Reward can change for new farmers and nodes over time'. why: once you started a node at a certain cpr_reward this will number will stay the same for 60 month

**Review by andhartl — COMMENTED** [review]
no longer true: 
image
TFTS will be used to pay for support requests.

**Review by andhartl — COMMENTED** [review]
up to 10 times less energy
image

**Review by andhartl — COMMENTED** [review]
vesting tft for Planet Friendly farmers only
image

**Review by andhartl — COMMENTED** [review]
Should say Planet Positive farming
image

File: src/farming/cpr_reward_current.md [hunk 1]
```diff
@@ -1,31 +1,14 @@
...
-  - Example, it could be logical that as the grid grows the cultivation nr can go up and farming nr can go down.
-  - We believe in the power of a "human blockchain" to define the right reward for new farmers.
-
-### CPR Calculations
-
-Calculating your 3Node's CPR can be done through the [Farming Hardware Calculator](farming_hardware_calculator)
-
-!!!include:farming_logic_disclaimer
\ No newline at end of file
+Farmed TFTs will go to a staking mechanism and only unlock if +30% of the 3Node's capacity is used.
```
Comment (andhartl) [inline src/farming/cpr_reward_current.md:14]: staking I sonly applicable for Planet Friendly farmers

File: src/farming/farming_intro.md [hunk 2]
```diff
@@ -1,37 +1,57 @@
...
 
-When a farmer connects a 3Node (server) to the ThreeFold_Grid, the total capacity of this server is detected by the ThreeFold's operating system - Zero-OS, and registered on the blockchain database. We call this proof-of-capacity. By keeping the server online, the farmer will be rewarded with [ThreeFold Tokens](token_what) (TFT) by the ThreeFold Foundation. We call this tf_farming.
+Farmers earn [ThreeFold Tokens](token_what) (TFT) for connecting Internet capacity in two ways:
+- Farming: When connecting capacity and keeping it online, a Farmer gets rewarded on a monthly basis by the ThreeFold Foundation.
+- Cultivation: When users reserve and use the provided Internet capacity to store their data or run IT workloads.
 
-Once 3Nodes are connected to the internet they become part of to the ThreeFold_Grid. Users can now deploy IT Workloads onto the TF_Grid. This process is known as tf_cultivation.
+## A circular economic model
+
+![](img/circular_tft.png)
```
Comment (andhartl) [inline src/farming/farming_intro.md:15]: needs update. does not include burning.

File: src/farming/farming_intro.md [hunk 3]
```diff
@@ -1,37 +1,57 @@
...
 ## Proof-of-Capacity
 
-The proof-of-capacity records three resources from the 3node (server):
+The proof-of-capacity records three Internet resources from the 3node (server):
 
 - Compute Capacity (CPU)
 - Memory Capacity (RAM)
 - Storage Capacity (SSD/HDD)
 
 The proof-of-capacity is stored on a distributed storage system and hashed (fingerprinted). The hashes verify the authenticity of the report and are stored in the minting transactions on the blockchain. 4 days later, the TFTs are transferred to the farmer. The minting consensus needs to be achieved before minting can actually happen.
```
Comment (andhartl) [inline src/farming/farming_intro.md:39]: 4 days after the end of the month .....

File: src/farming/farming_intro.md [hunk 4]
```diff
@@ -1,37 +1,57 @@
...
+- +1 Billion people to go digital in the next 5 years.
+
+The ThreeFold Grid offers the perfect infrastructure to supply the increasing Internet demand. Learn more about the ThreeFold Grid [here](grid_home).
 
 ## More Info
 
-- [How to become a Farmer](become_a_farmer) - A guide towards all the information needed to become a farmer.
-- [The ThreeFold Token](token_what) - An introduction to the currency of the new Internet.
+- [Different types of Farming](certified_farming)) - A guide towards the different types of farming.
+- [Farming Rewards](farming_logic3) - An introduction to the logics behind farming rewards.
```
Comment (andhartl) [inline src/farming/farming_intro.md:54]: need to check if linked doc is not about certified farming

File: src/farming/farming_logic3.md [hunk 5]
```diff
@@ -2,17 +2,20 @@
...
-TFT are rewarded when new capacity (storage & compute) is added to the ThreeFold_Grid in the form of hardware.
+TFT is rewarded to farmers when capacity (storage & compute) is made available to the ThreeFold_Grid by connecting a 3Node.
 
-This is done by [TFT Farming](become_a_farmer), ThreeFolds more sustainable equivalent of cryptocurrency miners.
+This is done by [Farming TFT](become_a_farmer), ThreeFold's sustainable equivalent of crypto mining.
 
-The amount of TFT which need to be farmed are calculated once a month by the open source [consensus driven token reward engine](tftech:consensus3).
+The amount of farmed TFTs is calculated monthly by the open source [consensus driven token reward engine](tftech:consensus3).
 
 !!!include:farming_logic_disclaimer
```
Comment (andhartl) [inline src/farming/farming_logic3.md:12]: consensus driven is not implemented yet. we should state this here!

File: src/farming/farming_logic3.md [hunk 6]
```diff
@@ -22,56 +25,25 @@ The specs of the farmers' 3Node:
 - Memory Capacity (RAM)
 - Storage Capacity (SSD/HDD)
 
-The proof of capacity is stored on a redundant storage system and hashed (fingerprinted). The hashes verify the authenticity of the report and are stored in the minting transactions on the blockchain. 4 days later, the TFTs are transferred to the farmer. The minting consensus needs to be achieved before minting can actually happen. In the transaction message, the hash will be stored on the blockchain and will represent an immutable proof of the capacity for each minting operation.
+The proof of capacity is stored on a redundant storage system and hashed (fingerprinted). The hashes verify the authenticity of the report and are stored in the minting transactions on the blockchain. 4 days later, the TFTs are transferred to the farmer.
```
Comment (andhartl) [inline src/farming/farming_logic3.md:29]: 4 days after month end ....

File: src/farming/farming_logic3.md [hunk 7]
```diff
@@ -22,56 +25,25 @@ The specs of the farmers' 3Node:
...
-
-> - The TFT farmed = result of compute,storage capacity connected to internet and having good enough uptime
-> - The TFT are staked per 3Node as arranged by TF_CHAIN. DIY Farmers can take tokens out of the staking pool once the 3node has 30% (\*) of its capacity used. Certified farmers can at free will take tokens out of the staking pool (starting Sept 2021).
-> - The Farmer gets 100% of farming (minting of TFT), in case of certified farming can be less.
-> - The Farmer gets 10% from TF_Cultivation (people buying IT capacity from TFGrid which is network of 3Nodes).
-
-For Certfied farming a contract is made between the farmer and Threefold (or TFTech). This can result in a certain part of the Farming Reward to be given to Threefold or TFTech as a license and/or support fee.
-
-To simulate yourself how farming is rewarded see **[farming calculator](farming_calculator)**
+The Consensus_Engine measures uptime and other SLA parameters. If the required minimal levels are not reached no farming rewards will be rewarded to the farmer.
```
Comment (andhartl) [inline src/farming/farming_logic3.md:38]: future

File: src/farming/farming_logic3.md [hunk 8]
```diff
@@ -22,56 +25,25 @@ The specs of the farmers' 3Node:
...
 ## Cultivation Fee and Burning of TFT
 
-10% of Cultivation fee goes to farmer (see above).
-
-20-40% of the cultivation fee is being burned by means of the TF_CHAIN in TFGrid 3.0. This means these tokens are destroyed. This makes sure that the amount of tokens being created are low in relation to the value increase of the grid.
-
-If a box is used for +80% then there will be more tokens burned than farmed, which means the amount of TFT goes down leading to more scarcity.
+- 10% of the cultivation fee goes to the farmer.
+- 20-40% of the cultivation fee is burnt by the the Tf_chain in TFGrid 3.0. The burning process destroys the tokens. This mechanism ensure that the amount of tokens being created are low in relation to the value increase of the grid.
+- If a node has +80% utilization, then more tokens will be burned than farmed to ensure scarcity of TFT.
```
Comment (andhartl) [inline src/farming/farming_logic3.md:44]: its actually not burning, its more staking tft for future farming rewards

File: src/farming_calculator/farming_calculator.md [hunk 9]
```diff
@@ -2,35 +2,15 @@
...
-- CPR = Cloud Production rate, which is a nr which can be calculated per server (3node).
-  - CPR expresses the capability of a server in relation to production of IT (compute/storage) capacity.
-- CPR_Reward = is the reward for being a farmer and is staked on the 3Node until 30% utilization of the capacity has been achieved.
-  - The TF_Council looks over the CPR reward and adjusts it over time in line with TFGrid growth and token price. Once rewarded to a farmer the CPR Reward does not change.
+Thanks to our sales model you also do not have to sell the capacity of your server(s), the threefold_approach takes care of this.
 
 ## Risks
 
-- Utilization does not go to 30% which means farming of tokens does not happen.
+- Utilization does not go to 30% which means all your farmed tokens remain staked.
```
Comment (andhartl) [inline src/farming_calculator/farming_calculator.md:13]: only for Planet Friendly farmers

File: src/farming_calculator/farming_calculator.md [hunk 10]
```diff
@@ -39,7 +19,7 @@ The big disruption ThreeFold brings is that we commodify this model, you don't h
   - _Mitigation: The cloud business by itself is valuable_
   - _In this case ThreeFold_Council would change the business model to more cultivation_
 - Digital Currencies become illegal
-  - _Mitigation: This is not really possible because the TFT is a real utility token, represents IT capacity._
+  - _Mitigation: This is not really possible because TFT is a real utility token, representing IT capacity._
```
Comment (andhartl) [inline src/farming_calculator/farming_calculator.md]: are we sure about tft being a utility token?

File: src/sidebar.md [hunk 11]
```diff
@@ -13,8 +13,7 @@
   - [More Information](grid_more_info)
 - [Farming](farming_home)
   - [What Is Farming](farming_intro)
-  - [Become a Farmer](become_a_farmer)
-  - [Certified Farming](certified_farming)
+  - [Farming Types](certified_farming)
```
Comment (andhartl) [inline src/sidebar.md:16]: see if linked doc talks about Planet Positive farming

File: src/sidebar.md [hunk 12]
```diff
@@ -13,8 +13,7 @@
...
   - [Farming Logic v3](farming_logic3)
   - [Farming Calculator](farming_calculator)
```
Comment (andhartl) [inline src/sidebar.md:18]: add her new hw rewards simulator pls

**Comment by Sacha96** [comment]
Fixed your comments @andhartl. Will create new pull request

**Pull request merged**